config/routes.php

```diff
@@ -1,16 +1,18 @@
 <?php
 
 use Cake\Routing\RouteBuilder;
-use Cake\Routing\Router;
 
-Router::prefix('Admin', function (RouteBuilder $routes) {
-	$routes->plugin('Queue', function (RouteBuilder $routes) {
-		$routes->connect('/', ['controller' => 'Queue', 'action' => 'index']);
+return static function (RouteBuilder $outer_routes) {
```

Comment:
```suggestion
return static function (RouteBuilder $routes) {
```

Reply:
I first thought this would make it more clear which routes variable is where but since there are already "nested variables" with the same name it seems it doesn't matter 🤷🏻